pyanaconda/modules/payloads/source/harddrive/harddrive.py

```diff
@@ -62,6 +64,10 @@ def get_state(self):
 
     def process_kickstart(self, data):
         """Process the kickstart data."""
+        if data.harddrive.biospart:
+            raise KickstartParseError("biospart support is not implemented",
```

Comment:
Please, translate the string and provide more information. It is not clear that you are talking about the `harddrive` command and its `--biospart` option.